src/lib/routes/admin-api/project/api-token.ts

```diff
@@ -116,8 +125,12 @@ export class ProjectApiTokenController extends Controller {
                 openApiService.validPath({
                     tags: ['Projects'],
                     operationId: 'deleteProjectApiToken',
+                    summary: 'Delete a project API token.',
+                    description:
+                        'Endpoint that allows deletion of a project API token.',
```

Comment:
Do you specify the token in the URL? And if it doesn't exist, is it a 200? Or a 404?

```suggestion
                        'This operation deletes the API token specified in the request URL. If the token doesn't exist, returns an OK response (status code 200).',
```